accel/tcg/cputlb.c

```diff
@@ -1751,7 +1751,7 @@ static inline uint64_t cpu_load_helper(CPUArchState *env, abi_ptr addr,
     oi = make_memop_idx(op, mmu_idx);
     ret = full_load(env, addr, oi, retaddr);
 
-#if defined(CONFIG_MIPS_LOG_INSTR)
+#if defined(CONFIG_CHERI_LOG_INSTR)
```

Comment:
Ideally this should not be CHERI specific, maybe rename it to `CONFIG_TCG_LOG_INSTR` ?

Reply:
I agree, I renamed it very early in the patch and should be changed. Will do.